recipes/libzippp/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibZipppConan(ConanFile):
+    name = "libzippp"
+    description = "A simple basic C++ wrapper around the libzip library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ctabin/libzippp"
+    license = "BSD-3-Clause"
+    topics = ("conan", "zip", "libzippp", "zip-archives", "zip-editing")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("zlib/1.2.11")
+        self.requires("libzip/1.7.3")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        # eg. 'libzippp-libzippp-v4.0-1.7.3'
+        extracted_dir = self.name + "-" + self.name + "-v" + self.version + \
+            "-" + self.deps_cpp_info["libzip"].version
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.replace_in_file('source_subfolder/CMakeLists.txt',
+                              'find_package(LIBZIP MODULE REQUIRED)',
+                              'find_package(libzip REQUIRED)')
+        tools.replace_in_file('source_subfolder/CMakeLists.txt',
+                              'libzip::zip', 'libzip::libzip')
```

Comment:
Will break as soon as https://github.com/conan-io/conan-center-index/pull/2972 will be merged. Please wait this PR, or inject `${CONAN_LIBS}`